store.go

```diff
@@ -582,6 +593,7 @@ type store struct {
 	autoNsMaxSize   uint32
 	graphDriver     drivers.Driver
 	layerStore      LayerStore
```

Comment:
Name this `rwLayerStore`? Possibly document the fields, even if they are “a private cache for $method”.